docs/GettingStarted.md

```diff
@@ -75,6 +75,7 @@ This will build the following binaries:
 * `sample_ext_app.exe`: A sample application for testing the sample extension driver.
 * `xdp_tests.exe`: Application for testing various XDP functionalities.  This requires the EbpSvc service to be running,
                 and the EbpCore and NetEbpfExt drivers to be loaded on a remote system to test.
+* `socket_tests.exe`: Application for testing the hook for the BPF_CGROUP_SOCK_ADDR program type and related attach types.
```

Comment:
```suggestion
* `socket_tests.exe`: Application for testing the BPF_CGROUP_SOCK_ADDR program type and related attach types.
```
("hook" should be plural, but simplified sentence instead of just making it be plural)